package.json

```diff
@@ -19,44 +19,67 @@
     "index.js"
   ],
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
-    "start": "ember server",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "lint:js:fix": "eslint . --cache --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "ember-cli-babel": "^7.23.1",
+    "ember-cli-htmlbars": "^5.3.2",
```

Comment:
Can you double check to see if we have any `addon/**/*.hbs` files? If not, can you move this back to devDeps?

Reply:
Fixed. I wasn't sure if the fact that it provides a helper meant that it should have an `ember-cli-htmlbars` dependency.

Reply:
Nope, you only need it as a dependency if you have handlebars files.